data/hails.txt

```diff
@@ -2030,7 +2030,288 @@ phrase "friendly author"
 
 phrase "friendly disabled"
 	word
-		"Our ship has been disabled! Please come board our ship and patch us up!"
+		"Our ship has been disabled!"
+		"We've gotten ourselves in a bit of a situation!"
+		"We've lost control of our ship!"
+		"Our ship took quite the pounding!"
+		"Hey!"
+		"S.O.S! S.O.S!"
+	word
+		" "
+	word
+		"Please come board our ship and patch us up!"
+		"We would appreciate any help getting us running again!"
+		"We can't fix this on our own!"
```

Comment:
not really sure if this line is... good (as in, it sounds impolite in some combinations)
imagine "Hey! We can't fix this on our own!", and technically they don't even ask for help at all
personally I'd remove this line